📖clusterctl based quick-start (#2590)
kubernetes-sigs/cluster-api: docs/book/src/clusterctl/overview.md

# Overview of clusterctl

The `clusterctl` CLI tool handles the lifecycle of a Cluster API [management cluster].

The `clusterctl` user interface is specifically designed for providing a simple "day 1 experience" and a
quick start with Cluster API; it automates fetching the YAML files defining [provider components] and installing them.

Additionally it encodes a set of best practices in managing providers, that helps the user in avoiding 

> mis-configurations or in managing day 2 operations like e.g upgrades.

Reviewer: ```suggestion
mis-configurations or in managing day 2 operations such as upgrades.
```
